js/dataframe/Example.svelte

```diff
@@ -1,53 +1,12 @@
 <script lang="ts">
-	import { csvParseRows, tsvParseRows } from "d3-dsv";
-	import type { Gradio } from "@gradio/utils";
-
-	export let gradio: Gradio;
 	export let value: (string | number)[][] | string;
-	export let samples_dir: string;
 	export let type: "gallery" | "table";
 	export let selected = false;
 	export let index: number;
 
 	let hovered = false;
 	let loaded_value: (string | number)[][] | string = value;
 	let loaded = Array.isArray(loaded_value);
-
```

Comment:
If a string CSV path is passed for examples for `gr.Dataframe`, the backend reads the file and sends the head as an array (in `gr.Dataframe.process_example()`), so this code is never used.